Add files for Update Manager functionality

simplycomputing opened the pull request:
### All Submissions:

* [x] Have you followed the [ClassicCommerce Contributing guideline](https://github.com/ClassicPress-research/classic-commerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [repo standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

A file was added that handles the update manager functionality for updates that are served from classiccommerce.cc. 

### How to test the changes in this Pull Request:

1. Install the new file in the includes folder and add the call to it in classic-commerce.php on a test site.
2. Visit the test site and check for updates.
3. The current version is set at 1.0.1, so the test site should show it needs an update as shown:

![updates](https://user-images.githubusercontent.com/46998578/69491707-d95c5880-0eec-11ea-833e-f0697e928729.jpg)

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

<!-- Mark completed items with an [x] -->

### Changelog entry

Add file for Update Manager functionality for updates served from classiccommerce.cc.

simplycomputing commented:
I have done a second commit to add an images folder with an icon (svg) and a banner (jpg). These are used in the update windows as shown below:

![icon](https://user-images.githubusercontent.com/46998578/69493762-ceaebd00-0f06-11ea-9872-d59b48453ad3.jpg)

![banner](https://user-images.githubusercontent.com/46998578/69493763-d3737100-0f06-11ea-9e1c-c7e274fac2de.jpg)

bahiirwa commented:
> 1. Any specific reason for creating a new `images` folder instead of using existing `assets/images`?

No. Let’s keep one. We need to remove the WP.org folder too.

> 2. The `icon.svg` is full of Adobe crap and isn't compressed. This site https://vecta.io/nano will squish it down and remove any unnecessary stuff.
+1

> 3. With greatest of respect to _CodePotent_, I’m not a huge fan of ASCII text banners as in the `class-wc-update-client.php` file. Can we cut it?
If there is no advantage one over the other, then let’s kill it and keep bitmap images.

timbocode commented:
> If there is no advantage one over the other, then let’s kill it and keep bitmap images.

Sorry Laurence. I was referring to this in the PHP file...

![codepotent](https://user-images.githubusercontent.com/4199514/70376324-cb043880-18ff-11ea-9293-17d99076bf13.jpg)

bahiirwa commented:
Thank you for the clarity. I agree but Let’s leave a credit comment to author. 😁

timbocode commented:
That takes up a whopping 490 bytes that does...but OK.

That leaves 1. and 2. then, as well as removing wp.org. 

Need to create a new issue for the latter I think.

bahiirwa commented:
I agree with you. I meant, we need to do something like `/* credits: codepotent.com/plugin.....*/` @simplycomputing is better at this than I.

simplycomputing commented:
Code Potent is already credited four times in that header - don't see the need to add a fifth but will do so if you want. I agree that the ascii logo can go (I'm not a big fan either)

Have moved images into assets/images and changed the line of code to point to this I think. PLEASE CHECK CAREFULLY. Works on a test site for me, but I may have missed something.

The svg image has been compressed and changed using the site you referenced.

Have done a new commit to add these changes.
